scripts/git/release.sh

```diff
@@ -0,0 +1,138 @@
+
+#!/usr/bin/env bash -e
+VERSION=$1
```

Comment:
parameters by positions are hard to maintain long term. I would use the approach used here https://github.com/Azure/pcs-cli/blob/azure-iot-pcs-simulation/solutions/devicesimulation-nohub/single-vm/setup.sh#L27

Reply:
Opened a [bug](https://msazure.visualstudio.com/One/_workitems/edit/2342875) to fix this.